key.services.yml

```diff
@@ -2,7 +2,8 @@ services:
 
   key_manager:
     class: Drupal\key\KeyManager
+    arguments: ['@entity.manager', '@config.factory']
```

Comment:
Cool, thanks for the explanation. Do we need to update the documentation in this repo's readme.md file?